ext/node/errors.rs

```diff
@@ -92,6 +93,21 @@ pub fn err_invalid_package_target(
   generic_error(msg)
 }
 
+pub fn err_missing_declaration_file(
+  path: &Path,
+  package_json_path: &Path,
+  subpath: &str,
+) -> AnyError {
+  let msg = format!(
+    "[TS7016] Could not find a declaration file for subpath {} in {}. {} implicitly has an 'any' type.",
```

Comment:
I'm not sure we have any errors in this format like this. For example, here's another ts error:

![image](https://github.com/denoland/deno/assets/1609021/0f2c7ba8-b313-41a6-9545-368eed6f2077)

Also, this wouldn't be surfaced with code 7016 in the lsp. Maybe we should just document this with a comment that it's `7016` for the time being and then work on improving all this code to return with error codes in the future?